.github/workflows/canary-integration-test.yml

```
@@ -73,8 +73,19 @@ jobs:
           toolbox=$(kubectl get pod -l app=rook-ceph-tools -n rook-ceph -o jsonpath='{.items[*].metadata.name}')
           timeout 15 sh -c "until kubectl -n rook-ceph exec $toolbox -- ceph mgr dump -f json|jq --raw-output .active_addr|grep -Eosq \"(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\" ; do sleep 1 && echo 'waiting for the manager IP to be available'; done"
           mgr_raw=$(kubectl -n rook-ceph exec $toolbox -- ceph mgr dump -f json|jq --raw-output .active_addr)
+
           timeout 60 sh -c "until kubectl -n rook-ceph exec $toolbox -- curl --silent --show-error ${mgr_raw%%:*}:9283; do echo 'waiting for mgr prometheus exporter to be ready' && sleep 1; done"
 
+      - name: check s5cmd version in the toolbox image
+        run: |
+          toolbox=$(kubectl get pod -l app=rook-ceph-tools -n rook-ceph -o jsonpath='{.items[*].metadata.name}')
+          kubectl -n rook-ceph exec $toolbox -- s5cmd version  | grep -q ^v2.2.1  || {
+            echo " Error: the version of s5cmd version in the  toolbox is not the expected v2.2.1"
```

```suggestion
          kubectl -n rook-ceph exec $toolbox -- s5cmd version  | grep -q ^v2.2.1  || {
            echo " Error: the version of s5cmd version in the  toolbox is not the expected v2.2.1"
```

also, I'm not sure if we can directly do this when running a command inside the pod without interactive exec. Please check

---

@subhamkrai  wrote:
> also, I'm not sure if we can directly do this when running a command inside the pod without interactive exec. Please check

Good point! my local tests were with interactive, so I just added `-it`